lib/localenv/localenv.go

```diff
@@ -438,9 +311,9 @@ func (env *LocalEnvironment) OperatorService(opsCenterURL string, options ...htt
 
 // SiteOperator returns Operator for the local gravity site
 func (env *LocalEnvironment) SiteOperator() (*opsclient.Client, error) {
-	operator, err := env.OperatorService(
-		defaults.GravityServiceURL, httplib.WithLocalResolver(env.DNS.Addr()), httplib.WithInsecure())
-	return operator, trace.Wrap(err)
+	return env.OperatorService(defaults.GravityServiceURL,
+		httplib.WithLocalResolver(env.DNS.Addr()),
+		httplib.WithInsecure())
```

Comment:
I realize not in scope for this PR, but I really need to work on getting rid of these hardcoded insecure calls everywhere.